.changeset/silly-rivers-allow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/tsconfig': minor
```

Comment:
Is this not a major change as it will mean that consumers may have to change their code?

Reply:
discussed offline - it is breaking but `0.*.*` version changes can all be breaking, until you go to the first major (`1.0.0`).

however, we decided should just be v1 anyway, so will make it a major.